packages/client-test-browser/package.json

```diff
@@ -2,6 +2,19 @@
   "name": "@dataunions/client-test-browser",
   "version": "0.0.0",
   "description": "Data Union Client Browser Tests",
+  "private": false,
+  "publishConfig": {
+      "access": "public"
+  },
+  "repository": {
+      "type": "git",
+      "url": "git+https://github.com/dataunions/data-unions.git",
+      "directory": "packages/client-test-browser"
+  },
+  "bugs": {
+      "url": "https://github.com/dataunions/data-unions/issues"
+  },
+  "homepage": "https://github.com/dataunions/data-unions/tree/main/packages/client-test-browser#readme",
```

Comment:
(GH doesn't let me comment on the right line, but a few lines down:)

Author: Data Union DAO
License: Apache-2.0